pkg/controller/constraint/constraint_controller.go

```diff
@@ -153,9 +205,10 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 			if err := r.Update(context.Background(), instance); err != nil {
 				return reconcile.Result{Requeue: true}, nil
 			}
+			// removing constraint entry from cache
+			r.constraintsCache.deleteConstraintKey(constraintKey)
```

Comment:
Why is `reportMetrics` not set to true here?

Reply:
ugh, i thought i set this already, merge issue maybe

Reply:
LOL